etc/spack/defaults/packages.yaml

```diff
@@ -30,9 +30,11 @@ packages:
       java: [jdk, openjdk, ibm-java]
       jpeg: [libjpeg-turbo, libjpeg]
       lapack: [openblas]
+      mariadb-client: [mariadb-c-client, mariadb]
       mkl: [intel-mkl]
       mpe: [mpe2]
       mpi: [openmpi, mpich]
+      mysql-client: [mariadb-c-client, mariadb]
```

Comment:
@chissg it looks like you added mariadb as a possible provider here but the `mariadb` package does not have a `provides` for `mysql-client` (only for `mariadb-client`).

Reply:
Yes, the ability of the `mariadb` package to provide a `mysql`-compatible client API has not been verified by me. I will remove it immediately.